Doc/library/enum.rst

```diff
@@ -159,12 +163,28 @@ Data Types
 ----------
 
 
-.. class:: EnumType
+.. class:: EnumDict
+
+   *EnumDict* is used by *EnumType* to keep track of the enum member orders and prevent reusing the member names.
```

Comment:
I would start by saying what it is, not what it's used for.

Ideally the documentation should also say something about when you would use this and why. I don't know why you would use it though, so I can't help with writing that explanation.